src/components/Cluster/NewCluster/MasterNodes.tsx

```diff
@@ -0,0 +1,80 @@
+import styled from '@emotion/styled';
```

Comment:
would it make sense to have this in the UI folder given it's a not connected component?

on the other hand, I do like having related things together.

Reply:
Not sure if we have a rule for this yet, but I consider the `UI` folder suitable for reusable "UI" components. This component is not really reusable outside this context